reference-implementation/test/readable-stream.js

```diff
@@ -122,26 +122,6 @@ test('ReadableStream reading a stream makes ready and closed return a promise fu
   );
 });
 
-test('ReadableStream avoids waiting-to-waiting transitions when immediately reading', t => {
```

Comment:
A bit of a shame to lose this property, but I guess it was never terribly robust anyway (e.g. if multiple people were calling `read()` in their `ready` handler).